apps/nextra/pages/en/build/move.mdx

```diff
@@ -23,3 +23,5 @@ The following external resources exist to further your education:
 - [IMCODING Move Tutorials](https://www.imcoding.online/tutorials?tag=Aptos)
 - [Pontem Move Playground](https://playground.pontem.network/)
 - [Collection of nestable Move resources](https://github.com/taoheorg/taohe)
+
+A new Move compiler and language version is currently in early beta testing. If you are interested to play with it, check [this page](./move/tools/compiler_v2.mdx).
```

Comment:
Is there a sub directory `tools` for `compiler_v2.mdx`? Maybe it is a github page issue but I cannot see it from this page.

Reply:
Yes there is, at the given location. This links typically not work in PRs.